theblues/tests/test_charmstore.py

```diff
@@ -734,29 +719,6 @@ def test_debug(self):
             debug_data = self.cs.debug()
             self.assertEqual('all clear', debug_data['status'])
 
-    def test_fetch_macaroon_successful(self):
-        with HTTMock(fetch_macaroon_200):
-            results = self.cs.fetch_macaroon()
-            self.assertEqual('{"mymacaroon": "something"}', results)
-
-    def test_fetch_macaroon_not_found(self):
-        with HTTMock(fetch_macaroon_404):
-            with self.assertRaises(EntityNotFound) as cm:
-                self.cs.fetch_macaroon()
-        self.assertEqual(cm.exception.args, ('http://example.com/macaroon',))
-
-    def test_search_with_macaroon(self):
-        with HTTMock(search_200_with_macaroon):
-            self.cs.macaroons = "[macaroon1, macaroon2]"
-            results = self.cs.search('foo')
-            self.assertEqual([{'Id': 'cs:foo/bar-0'}], results)
-
-    def test_list_with_macaroon(self):
-        with HTTMock(search_200_with_macaroon):
-            self.cs.macaroons = "[macaroon1, macaroon2]"
-            results = self.cs.list()
-            self.assertEqual([{'Id': 'cs:foo/bar-0'}], results)
-
```

Comment:
Don't we still need to test that the given client and cookies are used in the expected way?